docs/developers/tooling/oracles/supra.mdx

```diff
@@ -4,7 +4,7 @@ image: /img/socialCards/supra.jpg
 ---
 
 [Supra](https://supraoracles.com/) is a novel, high-throughput oracle and 
-intraLayer: a vertically integrated toolkit of cross-chain solutions (data 
+intra-layer: a vertically integrated toolkit of cross-chain solutions (data 
```

Comment:
```suggestion
IntraLayer: a vertically integrated toolkit of cross-chain solutions (data 
```

IntraLayer is a Supra product: https://supra.com/training/courses/supra-introduction-to-intralayer/

Reply:
Done ! Thanks